bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "legislative-simulation",
+  "version": "0.0.0",
+  "homepage": "https://github.com/jtoney1/legislative-simulation",
+  "authors": [
+    "james <jatoney@hotmail.com>"
+  ],
+  "license": "MIT",
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
```

Comment:
If we bundle them, we don't have to rely on a 3rd party server, but in all likelihood, their CDN servers will have much more uptime than ours. There are just enough freak stories to scare me into downloading them (like malicious code on the jQuery CDN).

But like I said, it's something pretty simple to add in later. Let's now worry about it for now.